src/Workspaces/Core/Portable/Log/HistogramLogAggregator.cs

```diff
@@ -102,6 +103,18 @@ private int GetBucket(int value)
 
                 return bucket;
             }
+
+            /// <summary>
+            /// Writes out these statistics to a property bag for sending to telemetry.
+            /// </summary>
+            /// <param name="prefix">The prefix given to any properties written; if you want a period as a separator, it's
+            /// your responsibility to pass that.</param>
+            public void WriteTelemetryPropertiesTo(Dictionary<string, object?> properties, string prefix = "")
```

Comment:
would not be opposed to making prefix required

Reply:
I figured an event being fired that just is reporting a single set of these doesn't really need the prefix, but I guess yeah, you can still be explicit. OK changing.

Reply:
Changed.